rest-service/docker/entrypoint_config.sh

```diff
@@ -2,6 +2,16 @@
 
 /opt/rest-service/docker/prepare_secrets.sh
 
+if [ ! -e "/tmp/config.yaml" ]; then
```

Comment:
Having this as a hard-coded path feels wrong and like it could lead to interesting issues. Can we make this accept a passed in arbitrary?

Reply:
(issues such as lingering config.yamls or similar)

Reply:
sure why not